README.md

```diff
@@ -173,6 +174,16 @@ Kind: *Trusted Indication*.
 
 Errors:
 
+### `AccountClaim`
+Sends a claim verified to be from the `source` to the `destination` on the parachain. It has an `expiration` timestamp and a `scope` used to define the scope under which such a `claim` is made. A chain should only send such a claim if it is verified to have been intended to be sent by the `source`. It facilitates authentication across parachains.
+
+- `source: MultiLocation`
+- `destination: MultiLocation`
+- `expiration: Compact<u48>`: Timestamp of the claim expiration
+- `scope: Vec<u8>`
```

Comment:
how is this defined?

Reply:
For example if a claim was used to authorize another account to trade you would create a claim to the dex as destination and set the scope to the assets you would authorize a transfer for.